platform/ios/Integration Tests/MGLCameraTransitionTests.mm

```diff
@@ -277,7 +277,7 @@ - (void)testFlyToCameraInDelegateMethod {
 
                 // Now set another coordinate.
                 MGLMapCamera *camera = [MGLMapCamera cameraLookingAtCenterCoordinate:target2
-                                                                        fromDistance:altitude
+                                                                            altitude:altitude
```

Comment:
Why did we removed this test?

Reply:
One of our deprecated methods. We could add another, and silence the warning.

Reply:
I understand. Thank you for explaining.